Cargo.toml

```diff
@@ -17,8 +17,8 @@ rev = "0c0cae6fa22b751e7fd95d7ce7556ac6d7b7010e"
 
 [workspace.dependencies.soroban-spec]
 version = "0.8.0"
-git = "https://github.com/stellar/rs-soroban-sdk"
-rev = "090f5c88e8f8b67f4d02010665aedfea6ee43db3"
+git = "https://github.com/ahalabs/rs-soroban-sdk"
+rev = "14ce186845311c75ee9daab023837bbfd5f382a5"
```

Comment:
Does this fork need merging in? Or are we waiting for a new release here? or what?

Reply:
Yes, this is waiting on this PR https://github.com/stellar/rs-soroban-sdk/pull/871